SocketRocket/ProxyConnect.h

```diff
@@ -0,0 +1,6 @@
+#import <Foundation/Foundation.h>
+
+@interface ProxyConnect : NSObject
```

Comment:
Let's rename this class, so we are consistent and can avoid clashing with anyone who has `ProxyConnect` in their app. Suggested name: `SRProxyConnect` :)